lib/tapjoy/ldap/base.rb

```diff
@@ -44,6 +44,16 @@ def add(distinguished_name, attributes)
         return return_result
       end
 
+      def add_attribute(dn, attribute, value)
```

Comment:
we probably want to standardize the variable names here to distinguished_name vs dn.